File: LamaAPI/Lama.Domain/DTO/Photo/UploadPhotoResultDTO.cs
namespace Lama.Domain.DTO.Photo
{
    public class UploadPhotoResultDTO
    {
        public long Id { get; set; }
        public string BlobId { get; set; }
        public string Blob64Id { get; set; }
        public string Blob256Id { get; set; }
        public string OriginalBlobId { get; set; }
        public string SharedLink { get; set; }
        public bool IsDeleted { get; set; }
        public string Description { get; set; }
    }
}

File: WebClient/lama/src/app/models/Photo/uploadPhotoResultDTO.ts
import { LikeDTO } from '../Reaction/LikeDTO';
import { ImageTagDTO } from './imageTagDTO';
import { TextOnPhotoDTO } from './textOnPhotoDTO';

export interface UploadPhotoResultDTO {
  id: number;
  name: string;
  blobId: string;
  blob64Id: string;
  blob256Id: string;
  userId: number;
  originalBlobId: string;
  sharedLink: string;
  isDeleted: boolean;
  description: string;
  reactions: LikeDTO[];
  isDuplicate: boolean;
  tags: ImageTagDTO[];
  text: TextOnPhotoDTO;
}
